src/helpers/updateHierarchy.helper.js

```diff
@@ -46,7 +47,15 @@ class HierarchyService {
           }
         }
       };
-      console.log(option.data.request.data);
+      const logObject = {
+        msg: 'bulk Update Hierarchy',
```

Comment:
@santoshilimi don't hardcode message in the log. use `service/messageUtil.js` file to get the message.

Reply:
Resolved